schema/spacewalk/susemanager-schema.changes

```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
+Mon Sep 21 14:00:22 CEST 2020 - shirocco88
+
+- version 4.2.1-2
+- Schema upgrade for Debian with ARM64v8+ (cpuach=aarch64, cpuarch=arm64) architecture
```

Comment:
No, this is not right.

Just add your entry after on top of the file, then an empty line.

The headers and the version change are added by the Release Engineer (me, in this case) with `tito tag`.

Reply:
Or in other words, you just need:
```
- Schema upgrade for Debian with ARM64v8+ (cpuach=aarch64, cpuarch=arm64) architecture

```
On top of what the file already has on master.

Reply:
Perfect!